native/cocos/core/data/Object.cpp

```diff
@@ -59,6 +59,10 @@ CCObject::CCObject(ccstd::string name /* = ""*/)
 
 CCObject::~CCObject() = default;
 
+void CCObject::destruct() {
+    _name.clear();
```

Comment:
Does _objFlags also need to be reset to 0?
I think `destruct` method should reset all member variables in class.

Reply:
All right, _objFlags is an exception.